src/types/bytes.rs

```diff
@@ -67,6 +67,11 @@ impl PyBytes {
     /// Creates a new Python byte string object from a raw pointer and length.
     ///
     /// Panics if out of memory.
+    ///
+    /// # Safety
+    ///
+    /// This function dereferences the raw pointer `ptr`,
```

Comment:
I'm sorry for my unclear words, but I meant that 'let's note that `ptr` is a leading pointer of an array (or contiguous memory?) of which length is `len` by referencing https://doc.rust-lang.org/std/slice/fn.from_raw_parts.html#safety